### DISCUSSION TITLE
Peer to Peer Audio/Video Calling

### DISCUSSION THREAD

**Discussion by RemonAhammad** [libp2p/rust-libp2p#5042] [Q&A] [answered]
how can I create a peer-to-peer video calling app, for two devices after establishing a connection?

**Comment by thomaseizinger** [comment, marked as answer]
I don't think libp2p is a good fit for that. I'd suggest reading the collection of standards around WebRTC to understand the rationale that goes into video calling. In a nutshell: You want to favor latency over reliability which means using an unreliable transport like UDP. Today, libp2p's transport have a different set of tradeoffs.

https://github.com/algesten/str0m is a good library. it is not as feature-rich as libraries like pion but it has support for audio and video streams.
